test/details.test.js

```diff
@@ -121,6 +127,10 @@ describe('vaadin-details', () => {
       expect(toggle.getAttribute('aria-expanded')).to.equal('true');
     });
 
+    it('should set aria-controls on toggle button', () => {
+      expect(toggle.getAttribute('aria-controls')).to.equal(`vaadin-details-content-${getUniqueId()}`);
```

Comment:
Please consider using a RegEx here, same as in [vaadin-radio-group](https://github.com/vaadin/vaadin-radio-button/blob/ac80d5e6cac1b266c44da81bac79a1c13e930d15/test/radio-group.test.js#L541) tests.